src/arena/CreateSprintTemplate.js

```diff
@@ -143,19 +146,108 @@ function CreateSprintTemplate(props) {
     getSanityItems();
   }, []);
 
+  useEffect(() => {
+    getConfiguration();
+  }, []);
+
+  useEffect(() => {
+    handleChange();
+  }, [title, columns, handleChange]);
+
+  // pulls the /templates api and sets the missions to the templates
+  async function getConfiguration() {
+    let options = await RestAPI.get("pareto", "/templates");
+    setMissions(options.map((option) => option.title));
+  }
+  const minimumOptions =
+    columns.Morning.items.length +
+      columns.Workday.items.length +
+      columns.Evening.items.length >=
+    3;
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const handleChange = () => {
+    const templates = missions.filter((mission) => mission === title.trim());
+    let message;
+    if (
+      title === "" &&
+      (columns.Morning.items.length === 0 ||
+        columns.Workday.items.length === 0 ||
+        columns.Evening.items.length === 0)
+    ) {
+      message = "";
+    } else if (title.length < 4) {
+      message = "Name should be four characters or more.";
+    } else if (templates.length > 0) {
+      message = "This name is already taken";
+    } else if (minimumOptions === false) {
+      message = "Add at least 3 Options.";
+    } else {
+      message = "";
+    }
+    setError(message);
+  };
+
   return (
     <>
-      <h1>{I18n.get("createTemplate")}</h1>
-      <FormGroup controlId="fName" bsSize="large" style={{ width: 300 }}>
-        <ControlLabel>{I18n.get("editTemplateName")}</ControlLabel>
-        <FormControl
+      <h1
+        style={{
+          width: "auto",
+          marginBottom: "3rem",
+          fontWeight: 900,
+        }}
+      >
+        {I18n.get("createTemplate")}
+      </h1>
+      <FormGroup
+        controlId="fName"
+        bsSize="large"
+        style={{
+          width: "auto",
+          display: "flex",
+          flexDirection: "row",
+        }}
+      >
+        <ControlLabel style={{ marginRight: 25, paddingTop: 10 }}>
+          {I18n.get("enterTemplateName")}
+        </ControlLabel>
+        <TextField
+          color="success"
+          error={error}
+          required
+          helperText={error}
+          style={{ width: 300 }}
+          label={I18n.get("templateName")}
+          variant="outlined"
           value={title}
           onChange={(event) => setTitle(event.target.value)}
         />
+        <Button
+          disabled={(title === "" && minimumOptions === false) || error}
+          variant="gradient"
+          onClick={createTemplate}
+          style={{ marginLeft: 30, height: "4.8rem" }}
+        >
+          {I18n.get("create")}
+        </Button>
       </FormGroup>
-      <Button variant="gradient" onClick={createTemplate}>
-        {I18n.get("create")} Sprint
-      </Button>
+      <h2
+        style={{
+          marginTop: 50,
+          textDecoration: "underline",
+          fontWeight: 400,
+        }}
+      >
+        {I18n.get("dailyAchievements")}
```

Comment:
Nice, really good to implement this with an internationalization approach from the start. 

I saw you added the English translations to strings.ts - awesome. Just a heads up, if the options do not exist in the other languages, what gets returned by I18n is the name of the string that's trying to be retrieved. (So, if I change language to Spanish, dailyAchievements shows up as, well, 'dailyAchievements'...)

<img width="1059" alt="Screen Shot 2022-05-11 at 11 36 26 AM" src="https://user-images.githubusercontent.com/84106309/167890392-76091f87-fc3d-4ef8-8636-8398250a66ac.png">

I'll do a quick PR to add the new strings to the other languages after this one gets merged, so don't worry about changing anything - I just wanted to explain the quirks of how the localization works, for your future reference!

Reply:
Yeah, it's not ideal behavior right? Maybe there is a way to return the English string by default through some sort of check.

Reply:
Not necessary at all for this PR, but it's a big weakness of this internationalization approach. In general, we should separate each language into their own string file, and then combine them into one big object when we initialize the I18N object at mount/refresh.

Reply:
I'll take a look at the documentation, I feel like there _has_ to be a config option that sets English as the fallback if the phrase/property doesn't exist in the given language...